utils/url.tsx

```diff
@@ -107,20 +107,32 @@ function formattedError(id: string, message: string): React.ReactElement {
 export function validateChannelUrl(url: string): React.ReactElement[] {
     const errors: React.ReactElement[] = [];
 
+    let isDirectMessageFormat = false;
+    if (url.indexOf('__') > -1) {
+        const USER_ID_LENGTH = 26;
+        const userIds = url.split('__');
+        isDirectMessageFormat = userIds.length === 2 && userIds[0].length === USER_ID_LENGTH && userIds[1].length === USER_ID_LENGTH;
```

Comment:
0/5, but you could simplify this whole check into a regex with `/^.{26}__.{26}$/`

Reply:
Done @hmhealey - I still ended up using a constant for `USER_ID_LENGTH` though